2024/2024-08-12-report.org

```diff
@@ -0,0 +1,124 @@
+#+title:  Weekly report for 5th of August to 12th of August 2024
+#+author: Philip Herron, Pierre-Emmanuel Patry, Arthur Cohen
+#+date:   2024-08-12
+
+** Overview
+
+Thanks again to [[https://opensrcsec.com/][Open Source Security, inc]] and [[https://www.embecosm.com/][Embecosm]] for their ongoing support for this project.
+
+** Completed Activities
```

Comment:
ah, yes you're right. this section is just a simple call to the GitHub API for the list of pull requests (not issues) merged since a specific date. I'll add it if I can remember it!